openmc/data/neutron.py

```diff
@@ -671,19 +682,28 @@ def from_ace(cls, ace_or_filename, metastable_scheme='nndc'):
                          'cross section is given.'.format(mt))
                     continue
 
-                # Create summed reaction with appropriate cross section
+                # Don't duplicate fission cross sections
+                if mt == 18:
+                    continue
```

Comment:
Why is this necessary? I would have thought that if mt=18 was already present, the code wouldn't make it here by virtue of `if mt not in data` above.

Reply:
If MT=18 is *not* present for a fissionable material, then the partial fission MTs are. There were a few cases (e.g. U234 and U240) where we were double counting fission cross sections.